tools/opt/opt.cpp

```diff
@@ -359,6 +359,18 @@ Options (in lexicographical order):
                This pass looks for components of vectors that are unused, and
                removes them from the vector.  Note this would still leave around
                lots of dead code that a pass of ADCE will be able to remove.
+  --webgpu-mode
+               Turns on the prescribed passes for WebGPU and sets the target
+               environmet to webgpu0. Other passes may be turned on via
+               additional flags, but such combinations are not tested.
+               Using --target-env with this flag is not allowed.
+               This flag is the equivalent of setting the following flags:
+                 --target-env=webgpu0
+                 --eliminate-dead-functions
+                 --eliminate-dead-code-aggressive
+                 --eliminate-dead-const
```

Comment:
There is a way of getting the options for the optimization passes printed without hard coding the order here.  See how -O and -Os work.

Reply:
Done